namespaces/aushape.yml

```diff
@@ -11,23 +11,23 @@ namespace:
     description: >
       Audit event serial number
   - name: node
-    type: string
+    type: keyword
     description: >
       Name of the host where the audit event occurred
   - name: error
-    type: string
+    type: keyword
     description: >
       The error aushape encountered while converting the event
```

Comment:
I'm not sure if this should be a `keyword`. Sure, the set of possible error messages is limited. However, I think it might be difficult for people to have to search by complete error messages. What do you think?

Reply:
If it is more like a free text then yes `text` type would be better 😄 
I will change it.

Reply:
Thanks, @lukas-vlcek!

Reply:
@spbnick The issue is that in [released common data model 0.0.12](https://github.com/ViaQ/elasticsearch-templates/releases/tag/0.0.12) the `error` field was defined like this:

```json
    "error": {
      "doc_values": true,
      "index": "not_analyzed",
      "type": "string"
    }
```
Which is in fact `keyword`. We can change it to `text` going forward, **please let me know**. In this I will have to add an exception for this field to make tests pass.

Reply:
I think I just didn't think this through at the time. I think "text" would be better. The index would be very small, and useful for the users.

Reply:
@spbnick Done.